ui/perfherder/SelectorCard.jsx

```diff
@@ -46,6 +47,12 @@ export default class SelectorCard extends React.Component {
   }
 
   fetchRevisions = async selectedRepo => {
+    // if a user selects a new project/repo, we don't want them to
+    // be able to select revisions until that new data has returned
+    if (Object.keys(this.state.data) !== 0) {
```

Comment:
Shouldn't this be:

    if (Object.keys(this.state.data).length !== 0) {

With ``.length`` added?

Reply:
Oh, yes, that's what I meant - but this actually still works. I'll change it so it's clearer.